src/Handler/CurlFactory.php

```diff
@@ -75,6 +75,7 @@ public function release(EasyHandle $easy): void
 
         if (\count($this->handles) >= $this->maxHandles) {
             \curl_close($resource);
+            unset($resource);
```

Comment:
This is not needed, the `$resource` variable is automatically destroyed when the function return (which happens directly after this).

Reply:
Thank you. I will update this.